### PR TITLE
SNO-86-escape-search-text

### PR DESCRIPTION
**Pull request by zoldello**

### CODE REVIEW

**Comment by zoldello** [comment]
@hitz Yes, I tested and retested it after seeing your comment. The ENCODED app fails when there are two or more question marks in a search string. This reduces the number of the question marks to one. The solution is fine to me but it would be nice to get a second opinion. Is this a good enough solution or should this part of the ticket be held off until I get to the root cause of the failure?

File: .travis.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ cache:
 addons:
   apt:
     packages:
-    - oracle-java8-set-default
+    - oracle-java9-set-default
```
Comment (caseylitton) [inline .travis.yml]: revert before moving to merge ready

Comment (zoldello) [reply .travis.yml]: @caseylitton Done. Thanks for the catch.

**Comment by zoldello** [comment]
@caseylitton I do not see my comment to you earlier. I removed the java reference.

**Pull request merged**